src/content/docs/infrastructure/prometheus-integrations/integrations-list/cockroach-db-integration.mdx

```diff
@@ -0,0 +1,83 @@
+* Supports strongly-consistent ACID transactions
+* Provides a familiar SQL API for structuring, manipulating, and querying data
+
+CockroachDB generates detailed time series metrics for each node in a cluster. [Monitor CockroachDB with Prometheus](https://www.cockroachlabs.com/docs/v22.1/monitor-cockroachdb-with-prometheus.html#before-you-begin) shows you how to pull these metrics into Prometheus.
+
+Use New Relic to monitor:
+* Cluster health
+* Node health
+* Store and disk capacity
+* SQL queries performance
+* High open file descriptor count
+* Certificate expirations
+
+## Enable the integration
+
+Follow these steps to enable the integration.
+
+1. Follow the [CockroachDB documentation](https://www.cockroachlabs.com/docs/v22.1/monitor-cockroachdb-with-prometheus.html) to add the required pod annotations for Prometheus to discover the metrics endpoints.
+
+2. Set up Prometheus monitoring. Prometheus metrics needs to be integrated with New Relic, you can use the Prometheus Agent or the Remote Write integration, see [how to send Prometheus metrics](/docs/infrastructure/prometheus-integrations/get-started/send-prometheus-metric-data-new-relic/).
+
+3. Use the following query to confirm metrics are being ingested as expected:
+
+  ```
+    FROM Metric SELECT count(*) WHERE job='cockroachdb' FACET metricName LIMIT MAX
+  ```
+
+4. Install the [CockroachDB quickstart](https://newrelic.com/instant-observability/?search=cockroachdb) to access built-in dashboards and [alerts](/docs/alerts-applied-intelligence/new-relic-alerts/learn-alerts/introduction-alerts/).
+
+  Once you imported, you can edit or clone the assets to adapt them to your specific requirements.
+
+## Find and use the data
+
+### Metrics
+
+Prometheus metrics are stored as dimensional metrics. You can [query using NRQL](/docs/telemetry-data-platform/get-data/apis/query-metric-data-type/) or use the [Data Explorer](/docs/query-your-data/explore-query-data/browse-data/introduction-data-explorer/) to browse the available metrics, facet, and filter by the associated dimensions.
+
+The different sets of metrics exposed by this integration are defined in the [CockroachDB documentation](https://www.cockroachlabs.com/docs/stable/monitoring-and-alerting.html).
+
+Use the following NRQL queries to understand the CockroachDB metrics being ingested in New Relic.
+
+<Callout variant="tip">
+Adjust the job name as per your configuration settings.
+</Callout>
+
+* List unique metric names:
+
+  ```
+  FROM Metric SELECT uniques(metricName) WHERE job='cockroachdb' LIMIT MAX
+  ```
+
+* Count number of metric updates:
+
+  ```
+  FROM Metric SELECT datapointcount() WHERE WHERE job='cockroachdb' FACET metricName
+  ```
+
+* Estimate data ingestion (daily ingest, in bytes):
+
+	```
+  FROM Metric SELECT bytecountestimate() WHERE job='cockroachdb' SINCE 1 day ago
+  ```
+
+## Troubleshooting
```

Comment:
some of the metrics are broken and we apply relabel config to fix them. We should point it out in the docs in case they are using remote write and not our solution

Reply:
We propose different queries depending if you are using Prometheus Agent or not